### PR TITLE
Circleci

### PR DESCRIPTION
**Pull request by Xorima**
Add circleci testing

Signed-off-by: Jason Field <jason@avon-lea.co.uk>

### Description

[Describe what this change achieves]

### Issues Resolved

[List any existing issues this PR resolves]

### Check List

- [ ] All tests pass. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD>
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] All commits have been signed for the Developer Certificate of Origin. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD>

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-* @chef-cookbooks/cookbook_engineering_team
+* sous-chefs/grafana
```
Comment (damacus) [inline .github/CODEOWNERS]: You want a new team for this in terraform?

File: .travis.yml [hunk 2]
```diff
@@ -1,39 +0,0 @@
...
-  - cookstyle --version
-  - foodcritic --version
-
-script: KITCHEN_LOCAL_YAML=kitchen.dokken.yml kitchen verify ${INSTANCE}
-
-matrix:
-  include:
-    - script:
-      - chef exec delivery local all
-      env: UNIT_AND_LINT=1
```
Comment (damacus) [inline .travis.yml]: We'll need to delete the webhook

File: .github/CODEOWNERS (hunk 1 above)
Comment (Xorima) [reply .github/CODEOWNERS]: Possibly, and might want  to name it keepalived lol

File: .travis.yml (hunk 2 above)
Comment (Xorima) [reply .travis.yml]: How do we do that? I have added a terraform file, will that do it?

File: .github/CODEOWNERS (hunk 1 above)
Comment (damacus) [reply .github/CODEOWNERS]: Open a PR and I'll import the resource

File: .travis.yml (hunk 2 above)
Comment (damacus) [reply .travis.yml]: It's on the settings tab... Which you don't have 🤦‍♂️because you're not on the maintainers

**Pull request merged**